readthedocs/oauth/services/github.py

```diff
@@ -57,21 +57,20 @@ def sync_organizations(self):
         try:
             orgs = self.paginate("https://api.github.com/user/orgs", per_page=100)
             for org in orgs:
-                org_details = self.get_session().get(org['url']).json()
-                remote_organization = self.create_organization(org_details)
+                org_details = self.get_session().get(org["url"]).json()
+                remote_organization = self.create_organization(
+                    org_details,
+                    create_relationship=True,
```

Comment:
Just making explicit that the relationship is for the user.

```suggestion
                    create_user_relationship=True,
```